app/controllers/miq_policy_controller.rb

```diff
@@ -534,32 +350,23 @@ def set_session_data
     session[:server_options]          = @server_options
   end
 
-  def features
-    [
-      {
-        :name     => :policy,
-        :title    => _("Policies"),
-        :role     => "policy",
-        :role_any => true
-      },
-    ].map { |hsh| ApplicationController::Feature.new_with_hash(hsh) }
-  end
-
   def breadcrumbs_options
     {
       :breadcrumbs  => [
         {:title => _("Control")},
         menu_breadcrumb,
       ].compact,
-      :not_tree     => %w[rsop export log].include?(action_name),
-      :record_title => :description,
+      :record_info    => @policy || @record,
+      :not_tree       => %w[rsop export log].include?(action_name),
+      :record_title   => :description,
+      # :include_record => action_name == 'show'
```

Comment:
Why is this commented?

Reply:
I will remove this comment, as this is no longer needed